docs/specs/multi-registry/multi_registry.md

```diff
@@ -0,0 +1,366 @@
+  the naming convention is not based on company names that are not necessarily unique and are complex to control.
+- **Name inconsistency**: By enforcing core policies, we minimize the risk of name inconsistency across registries.
+
+Alternatives:
+
+- We could make aliases optional in the imports section. To do so, we would need to rely on Weaver to automatically
+  detect entity IDs that are defined both in the local semantic convention file and the imported registry. When such
+  a conflict is detected, Weaver will report an error and ask the user to define an alias for the imported registry.
+  This approach could be supported in the future if the need arises.
+
+### Discussions
+
+#### Do we allow importing the same external registry with different aliases within the scope of a registry?
+- Short answer: No for consistency and readability.
+- Long answer: Although technically it is not a problem, it is a good practice to use the same alias for the same
+  registry inside the same file or across different files of the same registry. This practice makes the overall
+  registry more readable and maintainable. Weaver will report an error if different aliases are used for the same
+  registry in the scope of a single registry.
+
+#### Do we allow an alias defined in one semconv file to be visible in another semconv file?
+- Short answer: Yes, under certain conditions.
+- Long answer: An alias defined in a file containing `global_imports` section is visible in all the files of the
+  registry. Such an alias is called a global alias. A global alias is useful when the same imported registry is
+  used in multiple files within the same registry. In this situation, defining a global alias simplifies the
+  management of dependencies, as updating to a new registry version will only require a single change in the
+  registry file containing the global aliases. Weaver will report an error if an alias not defined locally or as
+  a global alias is used in a semconv file.
+
+#### Do we allow importing the same registry with different versions?
+- Short answer: No for consistency.
+- Long answer: Although technically it is possible to support multiple versions of an imported registry, it is a
+  good practice to align the versions inside the same registry. This practice make the overall registry more
+  consistent and easy to reason about. Weaver will report an error if different versions are used for the same
+  registry in the scope of a single registry.
+
+#### Do we allow reference of experimental entities across registries?
+- Short answer: No for backward compatibility.
+- Long answer: Experimental entities are not meant to be used in production. Experimental entities are subject to
+  change or removal without any notice in the next version of the registry. The type of an experimental entity can
+  be changed. Weaver will detect and report an error if an experimental entity is referenced across registries.
```

Comment:
I don't understand this limitation or reasoning behind it.

It's common to use experimental semconv in production, it's also common to implement them in OTel or outside it in the instrumentations.

It makes sense to require something that depends on experimental stuff to be experimental, but I don't understand why we should limit the ability to import experimental attribute/group.

Reply:
The approach was to support only the most essential/simple features at the beginning, with the option of adding such things later if the need arose. It seems that the need is already here :-)

Technically, I don’t think there’s a problem with supporting experimental definitions across registries. Even if we decide that this is supported by default, I believe we should, at a minimum, make it easier for users who don’t want to allow this. This could simply be implemented as an OTEL policy that prevents this type of dependency.